sourcestats.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
Needed to change the shell, because process substitution (`<()`) isn't POSIX compliant… Alternatively I'd need to figure out a way do this differently with `rh`. If `bash` (or `zsh`) isn't an option, could you give me a hint what `rh` is and maybe even a link to a man page? I only found https://github.com/twigly/rust-http-cli